src/main/jte/admin/edit_room_page.jte

```diff
@@ -37,7 +37,7 @@
 
             <br><br>
 
-            <input type="submit" class="input btn btn-primary" value="OK">
+            <input type="submit" class="input btn btn-primary" value="OK" aria-label="${context.localize("rooms.save")}" title="${context.localize("rooms.save")}">
```

Comment:
Instead of adding new labels, replacing the text of the button is probably nicer